app/models/task.py

```diff
@@ -1,5 +1,34 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    is_complete = db.Column(db.Boolean, nullable=True)
```

Comment:
I don't think you *need* two fields for this, one would do and it makes things more complicated to keep them in sync.  That said it doesn't hurt much.